recipes/torchsnapshot/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "torchsnapshot" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/torchsnapshot-{{ version }}.tar.gz
+    sha256: 618c4947ae500ee8750442fb5dc94f11899be2ddb6b829155c0948460cf94a4f
+  - url: https://raw.githubusercontent.com/pytorch/torchsnapshot/main/requirements.txt
+    sha256: 277517eab8cf57b752b42f2a62b8fec9cbc8423c49c006f08122e9035cc35a9a
+  - url: https://raw.githubusercontent.com/pytorch/torchsnapshot/main/dev-requirements.txt
+    sha256: a2d18efa1f120e0406d590b84ae9f6017112d1896d62677044baf9e7239474fe
```

Comment:
I'm assuming that these files are missing from the sdist above, right? If you can, please open an issue upstream about this.